backend/corpora/common/utils/regex.py

```diff
@@ -0,0 +1,7 @@
+USERNAME_REGEX = r"(?P<username>[\w\-\|]+)"
+UUID_REGEX = r"[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}"
+EXTENSION_REGEX = r"(?P<extension>h5ad)"
+DATASET_ID_REGEX = f"(?P<dataset_uuid>{UUID_REGEX})"
+COLLECTION_ID_REGEX = f"(?P<collection_uuid>{UUID_REGEX})"
+CURATOR_TAG_NAME_REGEX = r"(?P<tag>.*)"
```

Comment:
```suggestion
CURATOR_TAG_NAME_REGEX = r"(?P<tag>\.*)"
```

Reply:
Need to capture the entire tag but not extend into the extension, delimited by `.`

Reply:
Actually, nvm, I think the way it's being used in `REGEX` below will prevent that from happening 👍